players.js

```diff
@@ -4,18 +4,25 @@
  * @param {*} id
  * @param {*} index
  * @param {*} bot
+ * @param {*} unoCall
  */
-function player(deck, id, index, bot) {
+function player(deck, id, index, bot, unoCall) {
   this.isBot = bot;
   this.playerDeck = deck;
   this.playerID = id;
   this.playerIndex = index;
+  this.playerUnoCall = unoCall;
```

Comment:
You can just set this equal to 0 that way you don't have to pass in a parameter since Unocal just defaults to 0 on creation